src/app/models/image-reference.ts

```diff
@@ -11,7 +11,12 @@ export interface CustomImageAttributes {
     virtualMachineImageId: string;
 }
 
-export type ImageReferenceAttributes = MarketImageReferenceAttributes | CustomImageAttributes;
+export interface SharedImageGalleryImageAttributes {
```

Comment:
Isn't it the same as CustomImageAttributes then?

Reply:
yea Ill just have them share since its the same. both just use a vitrualmachineimageid.